README.md

```diff
@@ -156,14 +156,15 @@ For testing dialogs put this code into "pasta_eln/test.py":
 ``` Python
 import sys
 from PySide6.QtWidgets import QApplication
-from .dialogForm import Form
+from .GUI.form import Form
 from .backend import Backend
-from .communicate import Communicate
+from .guiCommunicate import Communicate
 
 app = QApplication(sys.argv)
 backend = Backend()
-doc = {'_id': 's-4cf85492d9684601b70e057c278e4ab5', '_rev': '1-3fbe417334a18b29e9f3180847dbae2b'}
-dialog = Form(backend, doc)
+comm = Communicate(backend)
+doc = backend.db.getDoc("m-3a43570c4fd84b1ab81a8863ae058fb0")
```

Comment:
Why do we use the identifier to load the document? Are we sure that this will be a consistent way of doing it, can this identifier be different various local installations?

Reply:
It is an example (hence in Readme) of how to open / stage a realistic form without going through the gui. It is easy for development to have a program to go right to the widget, without clicking 10 items. 
Since it is in Readme, it is not code for running